src/containers/my-courses/course-search-tool-bar/CourseCreateDescription.tsx

```diff
@@ -0,0 +1,51 @@
+import { ChangeEvent, FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+
+import AppTextField from '~/components/app-text-field/AppTextField'
+
+import { TextFieldVariantEnum } from '~/types'
+import { styles } from '~/pages/create-or-edit-lesson/CreateOrEditLesson.styles'
+
+interface CourseCreateDescriptionProps {
+  decriptionName: string
+  titleName: string
+  onChangeHandler: (e: ChangeEvent<HTMLInputElement>) => void
+  onChangeHandlerDesc: (e: ChangeEvent<HTMLInputElement>) => void
+}
+
+const CourseCreateDescription: FC<CourseCreateDescriptionProps> = ({
+  decriptionName,
+  titleName,
+  onChangeHandler,
+  onChangeHandlerDesc
```

Comment:
onChangeTitle and OnChangeDescription will be better

Reply:
done